JitsiConference.js

```diff
@@ -324,6 +324,9 @@ export default function JitsiConference(options) {
      * again by Jicofo.
      */
     this._videoSenderLimitReached = undefined;
+
+    this._firefoxP2pEnabled = browser.isVersionGreaterThan(109)
```

Comment:
Why not check if the browser is Firefox here too, thus simplifying further checks?

Reply:
Nope, it won't work because of the rejection logic at https://github.com/jitsi/lib-jitsi-meet/pull/2279/files#diff-b2f6195536d4c09eac93cd5482c9258cae0dc21129fca78a125fe545594eca30R3465 and https://github.com/jitsi/lib-jitsi-meet/pull/2279/files#diff-b2f6195536d4c09eac93cd5482c9258cae0dc21129fca78a125fe545594eca30R2172. The client will end up disabling p2p for the other browsers as well.

Reply:
Ack